src/boot/bootctl-install.c

```diff
@@ -191,14 +191,14 @@ static int version_check(int fd_from, const char *from, int fd_to, const char *t
         if (r == -ESRCH)
                 return log_notice_errno(r, "Source file \"%s\" does not carry version information!", from);
         if (r < 0)
-                return r;
+                return log_error_errno(r, "Failed to get version information of source file \"%s\": %m", from);
```

Comment:
hmm, get_file_version() itself is all over the place with log messages, some error codes are logged at LOG_DEBUG level, others above it.

So I guess this fix doesn't make things really worse, but it also doesn't clean up things properly, as it means: before we had some errors we wouldn't log about at all, and now we have some errors we'll log about twice...